charts/alertmanager/Chart.yaml

```
@@ -6,7 +6,7 @@ icon: https://raw.githubusercontent.com/prometheus/prometheus.github.io/master/a
 sources:
   - https://github.com/prometheus/alertmanager
 type: application
-version: 0.33.1
+version: 0.33.2
```

new feature should raise minor version.
```suggestion
version: 0.34.0
```